algo/src/test/java/com/hartwig/serve/sources/ckb/curation/CkbCuratorTest.java

```diff
@@ -47,6 +47,12 @@ public void canCurateFacilityNameBasedOnName() {
         assertEquals(expectedLocation, curator.curateFacilityName(location2));
     }
 
+    @Test
+    public void canCurateFacilityWithMissingCity() {
+        Location missingCity = createLocation(null, "facility", null);
+        assertEquals(missingCity, curator.curateFacilityName(missingCity));
+    }
```

Comment:
I suppose country could also be null, if you want to include that in the test.

Reply:
good one, added